src/commands/build_status_to_pagerduty.yml

```diff
@@ -0,0 +1,41 @@
+description: |
+  Send build status to PagerDuty integration.
+
+parameters:
+  integration-key:
+    description: Integration key (a.k.a routing key) provided by PagerDuty.
+    type: string
+  when:
```

Comment:
lets leave this to the caller? because maybe someone wants to run the command even if not on fail?